_includes/content/environment-file.md

```diff
@@ -19,33 +19,33 @@ Use the following guidelines and sample file to define the environment variables
  [Felix configuration reference]({{site.baseurl}}/reference/felix/configuration).
 {% endif %}
 
-
-For an etcdv3 datastore set the following
+For a Kubernetes datastore (default) set the following:
```

Comment:
Wondering if we want to make all the etcd / kdd differences start using tabs to simplify docs structure - WDYT?

Reply:
Added tab structure. Not sure where this shows up in docs to verify.

Reply:
Here's a preview @bmckercher123 : https://deploy-preview-4666--calico-master.netlify.app/getting-started/bare-metal/installation/binary-mgr